### DISCUSSION TITLE
Log format won't update after changing ConfigMap

### DISCUSSION THREAD

**Discussion by vitorfhc** [nginx/kubernetes-ingress#2889] [Q&A] [answered]
I found [this example](https://github.com/nginxinc/kubernetes-ingress/tree/v2.3.0/examples/custom-log-format), which I used to change the log formatting of my controller. After patching the configuration map, Kubernetes notified the pod of the changes.

```
Event(v1.ObjectReference{Kind:"ConfigMap", Namespace:"nginx", Name:"ingress-nginx...
```

But these changes didn't affect the logging. I am still getting the old log format. I restarted the pod, and it didn't work also.

Is there any other step I need to follow?

**Comment by jasonwilliams14** [comment]
@vitorfhc Hello.
You should be able to extend the logging of NGINX Ingress by just adding NGINX `variables` to the configMap. NGINX Ingress controller will automatically pick up the values that you want to add to your logging for you.

Could you paste your `configMap` and the output of `nginx -T` ?
We might to get more information, but this should get things started.

**Reply by vitorfhc** [reply]
Thanks for the help, but I just figured it out and added the answer to this post.

**Comment by vitorfhc** [comment, marked as answer]
Well, it was no more than just a stupid mistake.

Instead of doing `log-format:` in the ConfigMap, I did `log-format-upstream:`, and it worked.

**Reply by jasonwilliams14** [reply]
@vitorfhc Perfect. Glad you are good to go. Sometimes, it is the little things. :)
